lib/fastlane/plugin/fueled/helper/fueled_helper.rb

```diff
@@ -16,7 +16,7 @@ def self.nil_or_empty(input)
       # Returns the last tag in the repo, one can filter tags by specifying the parameter
       def self.fetch_last_tag(filter:)
         tag = nil
-        git_cmd = "git tag -l --sort=-creatordate"
+        git_cmd = "git tag -l --sort=-refname"
```

Comment:
`refname` or `version:refname`. 
not sure if the result is gonna be any different 🤔

Reply:
`refname`:
```
0.2.11
0.2.101
0.2.1
0.2.0
```

`v:refname`
```
0.2.101
0.2.11
0.2.1
0.2.0
```

I guess we need `v:refname` or `version:refname`, `refname` does a lexical sort, which could end up returning incorrect tags.